lib/ansible/modules/cloud/amazon/aws_direct_connect_link_aggregation_group.py

```diff
@@ -193,7 +193,7 @@ def lag_exists(client, lag_id=None, lag_name=None, verify=True):
             failed_op = "Failed to describe DirectConnect link aggregation groups."
         raise DirectConnectError(msg=failed_op,
                                  last_traceback=traceback.format_exc(),
-                                 exception=e)
+                                 response=e)
```

Comment:
Oh... I had intended to remove DirectConnectError and import it from ansible.module_utils.aws.direct_connect so the modules were sharing the code. Can you check if that fixes the problem? Nice catch.